core/src/main/java/com/digitalpebble/stormcrawler/parse/filter/MD5SignatureParseFilter.java

```diff
@@ -72,6 +93,11 @@ public void configure(Map stormConf, JsonNode filterParams) {
         if (node != null && node.isTextual()) {
             key_name = node.asText("signature");
         }
+        node = filterParams.get("keyNameCopy");
+        if (node != null && node.isTextual()) {
+            copyKeyName = node.asText("signatureOld");
+            System.out.println("HERE: signatureOld" + copyKeyName);
```

Comment:
Sorry, that was for debugging to see whether the test for a non-existent configuration property works. I wanted to avoid two properties (a boolean whether to copy the signature and the key name).